### PR TITLE
Refactor/certificates

### PR DESCRIPTION
**Pull request by alexscoelho**
Includes improvements to NewCertificate.js and certificates.js

### CODE REVIEW

File: src/app/views/certificates/certificates.js [hunk 1]
```diff
@@ -21,7 +18,7 @@ const statusColors = {
 };
 
 const Certificates = () => {
-  const [isLoading, setIsLoading] = useState(false);
+  const [isLoading] = useState(false);
```
Comment (alesanchezr) [inline src/app/views/certificates/certificates.js:21]: what is the point on having useState if you are never going to change its value? You can do just:
```
const isLoading = false;
```

or maybe you don't even need that variable?

File: src/app/views/certificates/certificatesRoutes.js [hunk 2]
```diff
@@ -2,19 +2,19 @@ import React from 'react';
...
   {
-    path: '/certificates/cohort/:cohortId',
+    path: "/certificates/cohort/:cohortId",
     exact: true,
-    component: React.lazy(() => import('./certificates')),
+    component: React.lazy(() => import("./certificates")),
   },
   {
-    path: '/certificates/new/:type',
+    path: "/certificates/new/:slug",
```
Comment (alesanchezr) [inline src/app/views/certificates/certificatesRoutes.js:15]: what do you mean slug? certificateSlug?

File: src/app/views/certificates/NewCertificate.js [hunk 3]
```diff
@@ -25,73 +20,61 @@ const NewCertificate = () => {
   const session = JSON.parse(localStorage.getItem('bc-session'));
```
Comment (alesanchezr) [inline src/app/views/certificates/NewCertificate.js:20]: use the `getSession` funcion I created on the redux session actions.

**Review by alesanchezr — CHANGES_REQUESTED** [review]

File: src/app/views/certificates/certificates.js (hunk 1 above)
Comment (alexscoelho) [reply src/app/views/certificates/certificates.js:21]: is being used, i leave the variable

**Pull request merged**